osu.Game/Screens/Play/Player.cs

```diff
@@ -190,6 +194,8 @@ private void load(AudioManager audio, OsuConfigManager config, OsuGameBase game)
 
             mouseWheelDisabled = config.GetBindable<bool>(OsuSetting.MouseDisableWheel);
 
+            storyboardEnabled = config.GetBindable<bool>(OsuSetting.ShowStoryboard);
```

Comment:
This shouldn't read from this setting solely, I'm pretty sure that'll break cinema mods from handling storyboard outros if this is disabled, since cinema mods ignore user settings.

At least use `DimmableStoryboard.ContentDisplayed`.